samples/modules/nanopb/src/simple.proto

```diff
@@ -0,0 +1,8 @@
+// A very simple protocol definition, consisting of only
+// one message.
+
+syntax = "proto2";
+
+message SimpleMessage {
```

Comment:
Can you use Zephyr's coding style here?

Reply:
You mean if one change it to `simple_message {` it would stop to work? Should it then be in Zephyr's tree?

Reply:
It wouldn't stop working, it only affects the generated source/header files, and for languages other than `C/C++` it would produce potentially weird syntax.

This only is a sample, so I wanted to keep it as close to the samples provided by google/nanopb, because I think it would be more common to look at those as a reference.